src/jdk.jpackage/share/classes/jdk/jpackage/internal/AppImageFile.java

```diff
@@ -59,11 +61,13 @@
 public final class AppImageFile {
 
     // These values will be loaded from AppImage xml file.
-    private final String creatorVersion;
-    private final String creatorPlatform;
+    private final Path appImageDir;
```

Comment:
What this field is for? This property doesn't belong to the class.

Reply:
It was used to show path to .jpackage.xml file in case of error from copyAsSigned(), but it is actually not needed, since copyAsSigned() should not produce invalid app image file.

Reply:
If not used, then I'd remove it.